xml/System/Type.xml

```diff
@@ -13644,10 +13644,10 @@ Types that are defined in the .NET Standard are not marked with <xref:System.Ser
         </Parameter>
       </Parameters>
       <Docs>
-        <param name="genericTypeDefinition">To be added.</param>
-        <param name="typeArguments">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="genericTypeDefinition">The generic type definition.</param>
+        <param name="typeArguments">An array of type arguments.</param>
+        <summary>Creates a generic signature type, which allows third party reimplementations of Reflection to fully support the use of signature types in querying type members.</summary>
+        <returns>A generic signature type.</returns>
         <remarks>To be added.</remarks>
```

Comment:
Should the remarks section be deleted?

Reply:
No, they should be left. Remarks that read "To be added." do not appear in the build.